instrument/Facilities.xml

```diff
@@ -481,7 +481,7 @@
 
 </facility>
 
-<facility name="ILL" zeropadding="6" FileExtensions=".nxs,.hdf,.inx,.asc">
+<facility name="ILL" zeropadding="6" FileExtensions=".nxs,.hdf,.inx,.asc" allowNoPrefix="True">
```

Comment:
These changes make sense but I'm not sure on the name of the attribute. How about `nofileprefix="True"`? The casing is more consistent (`FileExtensions` slipped through the net) and its slightly more direct about what it means.